commands/games/tictactoe.js

```diff
@@ -0,0 +1,145 @@
+const { Command } = require('discord.js-commando');
+const stripIndents = require('common-tags').stripIndents;
+
+const Currency = require('../../currency/Currency');
+
+const games = [];
+const combinations = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]];
+
+module.exports = class BlackjackCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'blackjack',
+			group: 'games',
+			memberName: 'blackjack',
+			description: 'Challenge someone to a game of tic-tac-toe for donuts!',
+			details: 'Challenge someone to a game of tic-tac-toe for donuts.',
+			guildOnly: true,
+			throttling: {
+				usages: 1,
+				duration: 30
+			},
+
+			args: [
+				{
+					key: 'member',
+					prompt: 'Which user would you like to challenge?',
+					type: 'member'
+				},
+				{
+					key: 'bet',
+					prompt: 'How many donuts do you want to bet?',
+					type: 'integer',
+					max: 1000,
+					validate: async (msg, bet) => {
+						bet = parseInt(bet);
+						const balance = await Currency.getBalance(msg.author.id);
+
+						if (balance < bet) {
+							return stripIndents`
+								You don't have enough donuts. Your current account balance is ${balance} 🍩s.
+								Please specify a valid amount of donuts.
+								`;
+						}
+
+						if (![100, 200, 300, 400, 500, 1000].includes(bet)) {
+							return 'Please choose one of 100, 200, 300, 400, 500, 1000 for your bet.';
+						}
+
+						return true;
+					}
+				}
+			]
+		});
+	}
+
+	async run(msg, args) {
+		const bet = args.bet;
+		const user = args.member;
+
+		const userBalance = await Currency.getBalance(user.id);
+
+		if (userBalance < bet) {
+			return stripIndents`
+				${user} doesn't have enough 🍩s. Their current account balance is ${userBalance} 🍩s.
+				Please try again later.
+				`;
+		}
+
+		if (games.indexOf(msg.guild.id) > -1) {
+			return msg.reply(`you can't start 2 games of Tic-Tac-Toe one the same server at once.`);
```

Comment:
Oh yea, I just made a mistake writing that sentence out thats all. I obv meant the *on